### PR TITLE
new ff README

### PR DESCRIPTION
**Pull request by Kully**

### CODE REVIEW

**Comment by Kully** [comment]
new branch to resolve failing tests

**Comment by Kully** [comment]
@cldougl ready to merge?

File: plotly/add_figure_factory_to_plotly_library.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+# Add A Figure Factory to the Plotly [Python Library](https://plot.ly/python/)
+
+## What is a Figure Factory?
+In the Python Plotly Library, we have very basic plot types that are created using the `plotly.graph_objs` module. These plot types include Scatter, Box and Bar types. They are the basis of the plots and charts instatiated by Plotly.
```
Comment (cldougl) [inline plotly/add_figure_factory_to_plotly_library.md]: - we have basic plot types that are created using the `plotly.graph_objs` module
- These plot types include Scatter, Box and Bar types. - you could link to the graph_objs file where it lists all the plot types

File: plotly/add_figure_factory_to_plotly_library.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    y = [4, 5, 6],
+    mode = 'markers',
+)
+
+data = [trace]
+
+py.iplot(data, filename='new-scatter-plot')
+```
+
+There is another type of chart which uses these basic plot types to make other types of graphs, and this is the figure factories. These are wrappers that utilize the code from `plotly.graph_objs` to build charts that can use their structures. A good example of a figure factory is the [Scatterplot Matrix](https://plot.ly/python/scatterplot-matrix/) as it utilizes `go.Scatter`, `go.Box` and `go.Histogram`.
```
Comment (cldougl) [inline plotly/add_figure_factory_to_plotly_library.md]: >this is the figure factories 
these are the figure factories

**Comment by cldougl** [comment]
💃

**Pull request merged**